src/Microsoft.Azure.NotificationHubs/Messaging/MessagingUtilities.cs

```diff
@@ -0,0 +1,83 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for 
+// license information.
+//------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.NotificationHubs.Messaging
+{
+    public static class MessagingUtilities
```

Comment:
Just curious if this class needs to be public or if these are only internal utilities.

Reply:
changed to `internal`